src/key_value_store.js

```diff
@@ -187,6 +183,14 @@ export class KeyValueStore {
     }
 }
 
+/**
+ * Helper to create a file-matching RegExp from a KeyValueStore key.
+ * @param key
+ * @returns {RegExp}
+ * @ignore
+ */
+export const getFileNameRegexp = key => new RegExp(`^${key}\\.[a-z0-9]+$`);
```

Comment:
Can we be sure the key doesn't contain characters that are not allowed in Regex? (e.g. a dot)